firehose.tf

```diff
@@ -0,0 +1,47 @@
+module "firehose_label" {
+  source  = "cloudposse/label/null"
+  version = "0.24.1"
+
+  attributes = ["firehose"]
+
+  context = module.this.context
+}
+
+resource "aws_s3_bucket" "firehose_bucket" {
+  count  = local.enabled && var.firehose_enabled ? 1 : 0
+  bucket = module.firehose_label.id
+  acl    = "private"
+}
+
+resource "aws_iam_role" "firehose_role" {
+  count = local.enabled && var.firehose_enabled ? 1 : 0
+  name  = module.firehose_label.id
+
+  assume_role_policy = <<EOF
```

Comment:
@Benbentwo  we try to not use HEREDOC.
Please change to using `data "aws_iam_policy_document" "assume_role"`

https://github.com/cloudposse/terraform-aws-eks-cluster/blob/master/iam.tf#L1